builder/tests/common-types/src/main/java/io/helidon/common/types/TypeNameBlueprint.java

```diff
@@ -197,12 +197,22 @@ default String declaredName() {
     }
 
     /**
-     * The fully qualified type name. This will include the generic portion of the declaration, as well as any array
+     * The fully qualified type name.
+     *
+     * @return the fully qualified name
+     */
+    default String fqName() {
+        // implemented by a custom method
+        return className();
+    }
+
+    /**
+     * The fully resolved type. This will include the generic portion of the declaration, as well as any array
      * declaration, etc.
      *
      * @return the fully qualified name which includes the use of generics/parameterized types, arrays, etc.
      */
-    default String fqName() {
+    default String resolved() {
```

Comment:
for consistency, this should be called `resolvedName()`, since `resolved` seems like it is a boolean typically.

Reply:
renamed